ergotree-interpreter/src/eval/context.rs

```diff
@@ -23,6 +25,8 @@ pub struct Context {
     pub inputs: Vec<IrBoxId>,
     /// Pre header of current block
     pub pre_header: PreHeader,
+    ///
+    pub var_map: HashMap<u8, Constant>,
```

Comment:
Could you make it  `extension: ContextExtension,` instead, please? We're going to need to manipulate multiple `ContextExtension` in the future, and to keep it consistent with Scala version (see `ErgoLikeContext`, our `Context` sister type there).